src/components/accordion/index.md.njk

```diff
@@ -110,6 +110,19 @@ To see the changes you've made, you may need to create a new 'session'. For exam
 
 Disabling controls is normally confusing for users. If there is no content for a section, either remove the section or, if this would be confusing for your users, explain why there is no content when the section is opened.
 
+## Use localisation to translate ‘Accordion’ into other languages
+
+Read how to [set up localisation with Nunjucks and Javascript](/get-started/localisation/).
```

Comment:
Change this line to remove reference to 'set up'. Maybe to 'Read how to use localisation with...'